### ISSUE TITLE
mod_rewrite is enabled but not working

### ISSUE THREAD

**Issue by matt-bailey** [puphpet/puphpet#1092]
I have a basic trusty VM and `mod_rewrite` is apparently enabled (I ran `sudo a2enmod rewrite` to check).

However, when I run the install script for a CMS we use (SilverStripe) it says "You need mod_rewrite to use friendly URLs with SilverStripe, but it is not enabled."

I've read about having to [allow overrides](http://askubuntu.com/questions/422027/mod-rewrite-is-enabled-but-not-working), but in the `/etc/apache2/sites-available` directory there are three files and I don't know which one to edit.

Two of them had `AllowOverride None` and the other had `AllowOverride All`. I changed them all to `AllowOverride All` and also added the extra line `Allow from All`. I then restarted apache, `sudo service apache2 restart`, but none of this made any difference.

Do you have any suggestions?

**Comment by althe3rd** [comment]
I don't know if this will solve your issue for sure but I also have been experiencing this problem with a Wordpress install and found that the default apache config (located at /etc/apache2/apache2.conf when using ubuntu) was set to allow override none even though my virtual host entries had allow override all.

As soon as I changed the allow override setting in apache2.conf and restarted apache (sudo service apache2 restart) I then saw that mod_rewrite started working.

The big question I have is how to make this a default setting in the config.yaml file as I have to re do that setting every time I were to create a new VM or config from puphpet.
